src/test/regress/expected/multi_size_queries.out

```diff
@@ -0,0 +1,159 @@
+--
+-- SIZE_QUERIES
+--
+-- Test checks whether size of distributed tables can be obtained with citus_table_size.
+-- To find the relation size and total relation size citus_relation_size and 
+-- citus_total_relation_size are also tested.
+CREATE TABLE nation_to_size (
+	n_nationkey integer not null,
+	n_name char(25) not null,
+	n_regionkey integer not null,
+	n_comment varchar(152));
+	
+-- Tests on non-distributed tables.	
+SELECT citus_table_size('nation_to_size');
+ERROR:  table "nation_to_size" is not distributed
+SELECT citus_relation_size('nation_to_size');
+ERROR:  table "nation_to_size" is not distributed
+SELECT citus_total_relation_size('nation_to_size');
+ERROR:  table "nation_to_size" is not distributed
+	
+SET citus.replication_model TO DEFAULT;
+SELECT create_distributed_table('nation_to_size', 'n_nationkey');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- Tests when the replication model is not streaming.
+SELECT citus_table_size('nation_to_size');
+ERROR:  cannot get the size
+DETAIL:  Currently, this function only works with distributed tables with streaming replication and reference tables.
+SELECT citus_relation_size('nation_to_size');
+ERROR:  cannot get the size
+DETAIL:  Currently, this function only works with distributed tables with streaming replication and reference tables.
+SELECT citus_total_relation_size('nation_to_size');
+ERROR:  cannot get the size
+DETAIL:  Currently, this function only works with distributed tables with streaming replication and reference tables.
+DROP TABLE nation_to_size;
+SELECT citus_table_size('nation_to_size');
+ERROR:  relation "nation_to_size" does not exist
+LINE 1: SELECT citus_table_size('nation_to_size');
+                                ^
+SELECT citus_relation_size('nation_to_size');
+ERROR:  relation "nation_to_size" does not exist
+LINE 1: SELECT citus_relation_size('nation_to_size');
+                                   ^
+SELECT citus_total_relation_size('nation_to_size');
+ERROR:  relation "nation_to_size" does not exist
+LINE 1: SELECT citus_total_relation_size('nation_to_size');
+                                         ^
+CREATE TABLE nation_to_size (
+	n_nationkey integer not null,
+	n_name char(25) not null,
+	n_regionkey integer not null,
+	n_comment varchar(152));
+SET citus.replication_model TO streaming;	
+SET citus.shard_replication_factor TO 1;
+SELECT create_distributed_table('nation_to_size', 'n_nationkey');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+\copy nation_to_size FROM '/Users/velioglub/Desktop/citus/src/test/regress/data/nation.data' with delimiter '|'
```

Comment:
I think this path is where data is stored in your computer :). When you use input/output directories for testing you should commit only those files, not the ones generated by make check